tests/test_int32_tensor.py

```diff
@@ -27,6 +27,19 @@ def test_pond(self) -> None:
                 out = z.reveal().eval(sess)
                 np.testing.assert_array_almost_equal(out, [4, 4], decimal=3)
 
+    def test_binarize(self) -> None:
+        x = Int32Tensor(tf.constant([2**31 + 2**31 + 3, 3], dtype=np.int32))
```

Comment:
can we add one more test with randomly generated values?  should be robust, feel like this test is useful but pretty specific